### ISSUE TITLE
1.5.0rc1: Previously added internet template connecting to wrong port.

### ISSUE THREAD

**Issue by MadDashMurphy** [GNS3/gns3-gui#1309]
The default GNS3 server port has been changed from 3080 to 8000.  When I attempt to view my template configuration for a previously added internet template (using core-linux-6.4-internet-0.1.img), GNS3 attempts to connect to 192.168.52.129:3080.

Not sure if this issue is also linked to now not being able to add a link to the internet template from inside a project.  This results in a warning "No port available.  Please configure this device".

**Comment by noplay** [comment]
The port has been change from 8000 to 3080.

If you are using ESXi or a remote servers you need to fix your config during the upgrade:
https://gns3.com/discussions/migration-to-1-5-for-esxi-users-

It's automated when using the builtin gns3vm support.

**Comment by MadDashMurphy** [comment]
Ah thanks.  Sorry I didn't spot it.  Is there something simple that also explains why GNS3 doesn't think that the GNS VM has started correctly too?  Maybe this is the same thing.  Oh and I should have said, I'm on VMware Workstation 12 Pro.

**Comment by noplay** [comment]
Do you have error?

**Comment by MadDashMurphy** [comment]
Not now thanks.

I did when I wrote this though.  Launching GNS3 would launch the VM (GNS3).  The VM would launch but the app would think that it hadn't finished.  Note that I've since followed your advice above, which worked and swapped the version of the GNS3 VM to 1.5, which allowed me to add the Internet for GNS3 VM appliance back again (https://gns3.com/qa/client-version-1-4-6-differs-wit).

I still can't get internet through the appliance and am confused as to why my SW1 switch gives a different subnet for each attached machine but I'll work through the community posts and post in the correct place if I can't find an answer.  Most of these little issues are probably due to me being new to GNS3.
